apps/builder/app/canvas/instance-selected.ts

```diff
@@ -233,8 +234,12 @@ const subscribeSelectedInstance = (
     }
     const activeStates = new Set<string>();
     for (const state of availableStates) {
-      if (element.matches(state)) {
-        activeStates.add(state);
+      try {
```

Comment:
could you please add comment some pseudo classes like :open or :current are not supported in .matches method